pyvista/plotting/qt_plotting.py

```diff
@@ -440,17 +460,45 @@ def __init__(self, parent=None, title=None, off_screen=None,
 
             self.iren.AddObserver("KeyPressEvent", self.key_press_event)
```

Comment:
Why not directly:

`self.iren.AddObserver("KeyPressEvent", super(QtInteractor, self).key_press_event)`

I don't think the `key_press_event_signal` is necessary. Is it to allow classes built on top of `QtInteractor` to catch it?

Reply:
We're learning through the ~~horror~~ magic of Azure about all the problems of not triggering events using signals.  It's messy, but as both `qt` and `vtk` want control of the main thread, when they fight we get a segfault.